### PR TITLE
Last one wins

### LINKED ISSUE #60: Define "last one wins" approach when Basic Card modifiers are equivalent
My understanding of the consensus that has emerged through discussion of PR API issue 684 [1] is that each specification should define its own approach to resolving duplicate instances of modifiers, and that payment method specifications should favor a "last one wins" approach. 

As I understand it, this is the approach taken by current implementations of Basic Card. 

I am raising an issue here so that we can close issue 684 on PR API more quickly (with an informative note in that specification). @marcoscaceres has indicated he would work on the text in Basic Card to capture the "last one wins" approach. However, we propose to wait until resolving issue #59 because the algorithm will be simpler if we drop supportedTypes.

[1] https://github.com/w3c/payment-request/issues/684#issuecomment-405612400

### PR DESCRIPTION
**Pull request by marcoscaceres**
closes #60 

The following tasks have been completed:

 * [ ] Confirmed there are no ReSpec errors/warnings.
 * [ ] Added Web platform tests (link)
 * [ ] added MDN Docs (link)

Implementation commitments:

 * [ ] Safari (link to issue)
 * [ ] Chrome (link to issue)
 * [ ] Firefox (link to issue)
 * [ ] Edge (public signal)

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/payment-method-basic-card/pull/69.html" title="Last updated on Mar 14, 2019, 11:09 AM UTC (11f4c11)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/payment-method-basic-card/69/29e3f10...11f4c11.html" title="Last updated on Mar 14, 2019, 11:09 AM UTC (11f4c11)">Diff</a>

### CODE REVIEW

**Review by ianbjacobs — COMMENTED** [review]
Thanks, @marcoscaceres! This seems ok to me, but I'd rather have an implementer look at the code closely.

**Comment by marcoscaceres** [comment]
Thanks, @ianbjacobs. I might get @danyao to take a look. 

@danyao, this is still quite drafty - I need to give it a bit more polish still (it even has some markup errors). However, if you want to take a quick look just as a first sanity check, I would appreciate that.

File: index.html [hunk 1]
```diff
@@ -458,6 +458,51 @@ <h4>
...
+        </ol>
+      </section>
+      <section>
+        <h2>Steps for applying modifiers</h2>
+        <p>
+          The steps for applying modifiers.
+        </p>
+        <ol class="algorithm">
+          <li>
+            Let <var>modifiers</var> be
```
Comment (danyao) [inline index.html:493]: This sentence seems incomplete?

**Review by danyao — COMMENTED** [review]
"Steps for showing the payment handler" looks good. From w3c/payment-request#684, it seems that Chrome may already behave this one, but I'll double check the code again (don't want to block you here).

"Steps for applying modifiers" seem incomplete. I assume it's also meant to make last modifier win, for consistency?

Shall we also add a WPT for this?

File: index.html (hunk 1 above)
Comment (ianbjacobs) [reply index.html:493]: They do so much, those modifiers! Please just let them be. Just this once!

Comment (marcoscaceres) [reply index.html:493]: heh, sorry about that @danyao ... yeah... put this up waaay too early. I was working on it a few weeks ago and forgot it was in such a bad state. Will try to fix it up soon.

**Comment by marcoscaceres** [comment]
Will send a new one soon.

**Pull request closed without merging**